server/dynamo.go

```diff
@@ -33,15 +33,7 @@ func (c *Server) InitDynamo() {
 	c.dynamo = svc
 }
 
-func (c *Server) fetchRedemptionV2(issuer *Issuer, ID string) (*RedemptionV2, error) {
-	issuerUUID, err := uuid.FromString(issuer.ID)
-	if err != nil {
-		c.Logger.Error("Bad issuer id")
-		return nil, errors.New("Bad issuer id")
-	}
-
-	id := uuid.NewV5(issuerUUID, ID)
-
+func (c *Server) fetchRedemptionV2(id uuid.UUID) (*RedemptionV2, error) {
```

Comment:
maybe a function comment stating that the caller should prepare the uuid identifier appropriately to the implementation? (uuidv5 in this case?)